Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:alpine AS build
+
+LABEL MAINTAINER = 'Friends of Go (it@friendsofgo.tech)'
+
+RUN apk add --update git
+WORKDIR /go/src/github.com/friendsofgo/graphiql
+COPY . .
+RUN TAG=$(git describe --tags --abbrev=0) \
+    && LDFLAGS=$(echo "-s -X main.version="$TAG) \
+    && go build -o /go/bin/graphiql -ldflags "$LDFLAGS" cmd/graphiql/main.go
+
+# Building image with the binary
+FROM alpine
```

Comment:
here there's no need to use `alpine`
I'd say that `FROM scratch` is more appropriate since you need an empty image to run the binary.

WDYT?

Reply:
Totally true!!!

As we explained in our [blogpost](https://blog.friendsofgo.tech/posts/dockerizando-tu-aplicacion-en-go/), our main idea was, precisely, having an image strictly with the binary.

@aperezg didn't do his job in the review process :-1: 
Fortunately, we've great colleagues like you! :smile: